src/main/resources/assets/securitycraft/models/item/reinforced_cobweb.json

```diff
@@ -0,0 +1,6 @@
+{
+    "parent": "item/generated",
+    "textures": {
+        "layer0": "securitycraft:item/reinforced_cobweb"
```

Comment:
Use the vanilla cobweb texture here (`block/cobweb`) and remove the one you added. The item tint will be applied nonetheless.

Reply:
Oh,  I didn't know that.